content/docs/gherkin/reference.md

```diff
@@ -99,7 +99,12 @@ You can write anything you like, as long as no line starts with a keyword.
 
 ## Rule
 
-The (optional) `Rule` keyword has been added in Gherkin v6. (Note that Gherkin 6 has not yet been incorporated into all implementation of Cucumber!)
+The (optional) `Rule` keyword has been part of Gherkin since v6. 
+
+{{% note "Cucumber Support for Rule"%}}
```

Comment:
This is nice; is there any way people can see if their implementation has implemented it yet?